_includes/cloud/enable-ssh.md

```diff
@@ -121,7 +121,7 @@ To add an SSH key using the Project Web Interface:
 
 1.	Copy your SSH public key to the clipboard.
 
-	If you don't already have SSH keys on that machine, see [GitHub documentation](https://help.github.com/articles/generating-an-ssh-key) to create them.
+	If you do not already have SSH keys on that machine, see [GitHub documentation](https://help.github.com/articles/generating-an-ssh-key) to create them.

```

Comment:
Initial cap on "do"

Reply:
I'm not sure what you mean in this specific case.